### DISCUSSION TITLE
Option to Add Numbers to Chapter Files

### DISCUSSION THREAD

**Discussion by Borlean** [audiamus/AaxAudioConverter#135] [Q&A] [answered]
Lovely piece of software and very helpful when combined with Book Lib Connect. 

Just wondering if it's possible to add an option to the Naming scheme to include track numbers instead of the numbers spelled out. 1, 2, 3 instead of one, two, three. 

With it spelled out, the files are not in numerical order where eight, eighteen and eleven are all next to each other. 

Unless I'm missing an option/drop down somewhere, this would help a lot. 

edit: spelling

**Comment by audiamus** [comment, marked as answer]
You are referring to "named chapters", I presume. Otherwise chapter numbers will always be strictly numerical. For named chapters, the file names are verbatim. So if the author says "Chapter One", the file name or directory name will also be "Chapter One". That is by intention.

What you can do here if your player gets confused:

- Play the book from the playlist. It's always created if there is more than one file to the book. And it always has the titles in correct order.
- Or, if your player does not support playlists, change the "Use named chapters" option to "Yes, always precede with number". It will produce slightly longer file names where alphanumeric sorting matches the chronological order.

**Comment by Borlean** [comment]
Interesting and good to know! I'm using AudioBookShelf which plays them in order as noted in your first bullet. 
But just so I understand my options, and if I use another player in the future that doesn't have that option, where is the setting for "Use named chapters"? I see a "Chapter Folders Name Prefix" option but that's greyed out (assuming because of the AAX file)

**Reply by audiamus** [reply]
In the "Basic Settings", under the classic Windows system menu, icon on the left of the title bar. There's a whole bunch of additional settings there. The manual will explain them.

**Comment by Borlean** [comment]
Much appreciated. Will review the manual. Thanks for the response. Marking Closed.
